db/migrate/20170928103153_fix_whitehall_about_page_first_published_at.rb

```diff
@@ -0,0 +1,22 @@
+require "csv"
+
+class FixWhitehallAboutPageFirstPublishedAt < ActiveRecord::Migration[5.1]
+  def up
+    update_count = 0
+    data = CSV.read(Rails.root.join("db", "migrate", "data", "about-first-published-at.csv"))
+    start_datetime = "2016-02-29 09:24:09"
+    end_datetime = "2016-02-29 09:24:11"
+
+    data.each do |content_id, first_published_at|
+      count = Edition
+        .with_document
```

Comment:
I don't think we need this line since we never access the document as part of a model instance.